### ISSUE TITLE
Problem getting font height

### ISSUE THREAD

**Issue by robinmonjo** [libharu/libharu#84]
Hello,

I need to restrict the text that can be written in a block. To do so, I use `HPDF_Page_MeasureText` to get the number of char that can be written in my rect. I also need to find out the height of my text so I can know the number of lines that I will be able to write (number of char * number of lines).

I tried to use `HPDF_Font_GetCapHeight` but this gave me a huge number (725 or something) , this is for sure not the height of the text I try to print.

To summarise, I know the font, the font size of my text and I need to know the height of it before printing it.

Any pointers ?
Cheers

**Comment by Afanas89** [comment]
Hello,
use <code>HPDF_Font_GetCapHeight * yourFontSize / 1000.0</code>
